From d178eae4e207931b038978db582db7e82ceb3f6f Mon Sep 17 00:00:00 2001
From: Davor Joja <davorjoja@logicbricks.com>
Date: Mon, 23 Jun 2014 19:58:29 +0200
Subject: [PATCH 0408/1566] gpu: drm: xylon: Removed unneeded define

Removed unneeded clock limit define since this is handled inside
connector mode_valid slave function call.

Signed-off-by: Davor Joja <davorjoja@logicbricks.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
---
 drivers/gpu/drm/xylon/xylon_connector.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/xylon/xylon_connector.c b/drivers/gpu/drm/xylon/xylon_connector.c
index a0b5cbc..ed38cca 100644
--- a/drivers/gpu/drm/xylon/xylon_connector.c
+++ b/drivers/gpu/drm/xylon/xylon_connector.c
@@ -27,8 +27,6 @@
 #include "xylon_connector.h"
 #include "xylon_drv.h"
 
-#define CLOCK_HIGH_LIMIT 165000
-
 struct xylon_drm_connector {
 	struct drm_connector base;
 	struct drm_encoder *encoder;
-- 
1.7.5.4

